tests/fsharp/SDKTests/tests/DefaultFSharpCore - IsNotPreviewSdkEnablePreviewCheck.proj

```diff
@@ -0,0 +1,21 @@
+<Project ToolsVersion="4.0" DefaultTargets="Test" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Verifies that in an RTM sdk with the previewcheck disabled it selects the already shipped fsharp.core package -->
```

Comment:
Fix this comment; it says preview check is disabled, but the test shows otherwise.